types/nginstack__datasource/OTHER_FILES.txt

```diff
@@ -0,0 +1 @@
+keys/MimeTypes.d.ts
```

Comment:
is the reason this isn't exported because people are expected to deep-import it like `import { APP_DATASOURCE } from '@nginstack/datasource/key/MimeTypes'`?

Reply:
That's precisely the reason